### PR TITLE
Fix dbsnp resource

### PR DESCRIPTION
**Pull request by jkgoodrich**

### CODE REVIEW

File: gnomad/resources/grch38/reference_data.py [hunk 1]
```diff
@@ -43,6 +43,16 @@ def _import_clinvar(**kwargs) -> hl.Table:
     return clinvar
 
 
+def _import_dbsnp(**kwargs) -> hl.Table:
+    dbsnp = import_sites_vcf(**kwargs)
+    dbsnp = dbsnp.group_by(dbsnp.locus, dbsnp.alleles).aggregate(
+        rsid=hl.agg.collect_as_set(dbsnp.rsid)
+    )
+    dbsnp = hl.split_multi(dbsnp, permit_shuffle=True)
```
Comment (ch-kr) [inline gnomad/resources/grch38/reference_data.py:51]: might be helpful to add that this `permit_shuffle` is set because the dbsnp vcf has duplicate loci

**Review by ch-kr — APPROVED** [review]
should update changelog, but lgtm

**Comment by jkgoodrich** [comment]
Note resource loading from current code here: gs://gnomad-public/resources/grch38/dbsnp/dbsnp_b154_grch38_all_20200514.ht if helpful for review

**Review by ch-kr — APPROVED** [review]
thanks for updating!

**Pull request merged**